src/database/config.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
   test: {
     username: process.env.DB_USER,
     password: process.env.DB_PASSWORD,
-    database: process.env.DB_NAME,
+    database: process.env.DB_NAME_TEST,
```

Comment:
I think some one is already working on resolving config errors? I could not test your feature until i returned this changes back to the way it was..... I suggest we leave the config file the way it is till the fix branch is merged

Reply:
Its good we have two databases, one for test and one for developement